src/game/reducers/update-player/index.ts

```diff
@@ -1,11 +1,13 @@
+import cloneDeep from 'lodash.clonedeep'
+
 import { IGame, IPlayer } from '../../types'
 
 export const updatePlayer = (
   game: IGame,
   playerId: IPlayer['id'],
   newPlayerProperties: Partial<IPlayer>
 ): IGame => {
-  const newGame = structuredClone(game)
+  const newGame = cloneDeep(game)
```

Comment:
This change is necessary to account for `jsdom` now being used for the tests (which is needed to support UI testing): https://github.com/jeremyckahn/farmhand-shuffle/pull/9/files?diff=split&w=1#diff-1e058ca1442e46581b13571fb8d261f9e1f5657e26c96634d4c1072f0f0347f1R7